lib/kernel/cuda/subgroup_shuffle.ll

```diff
@@ -0,0 +1,64 @@
+; ModuleID = 'subgroup_shuffle.bc'
+source_filename = "subgroup_shuffle.cu"
+target datalayout = "e-i64:64-i128:128-v16:16-v32:32-n16:32:64"
+target triple = "nvptx64-nvidia-cuda"
+
+; Function Attrs: convergent mustprogress nounwind
+define noundef i32 @_Z23intel_sub_group_shuffleij(i32 noundef %0, i32 noundef %1) local_unnamed_addr #0 {
+  %3 = tail call i32 @llvm.nvvm.shfl.idx.i32(i32 %0, i32 %1, i32 31)
```

Comment:
Are there no Clang builtins for these? .ll implementations are a maintenance burden.